src/Patch/X86_64/Layer2_X86_64.h

```diff
@@ -18,47 +18,48 @@
 #ifndef LAYER2_X86_64_H
 #define LAYER2_X86_64_H
 
+#include <memory>
+
+#include "llvm/MC/MCInst.h"
+
 #include "Patch/Types.h"
-#include "Patch/X86_64/RelocatableInst_X86_64.h"
+
+#include "Platform.h"
 
 namespace QBDI {
 
+class RelocatableInst;
+
 #ifdef QBDI_ARCH_X86_64
-#define movrr mov64rr
-#define movri mov64ri
-#define movmr mov64mr
-#define movrm mov64rm
-#define pushr push64r
-#define popr pop64r
-#define addri addr64i
-#define lea lea64
-#define popf popf64
-#define pushf pushf64
-#define jmpm jmp64m
+#define movrr(dst, src) mov64rr((dst), (src))
```

Comment:
I'm not fan of such macros (maybe constexpr functions)

Reply:
done